packages/replay-next/playwright/tests/utils/general.ts

```diff
@@ -160,6 +160,7 @@ export async function takeScreenshot(page: Page, locator: Locator, name: string)
 
   // Make sure any suspended components finish loading data before taking the screenshot.
   await awaitNoLoaders(page, locator);
+  await delay(1);
```

Comment:
What's this about?

Reply:
I noticed that sometimes dark screenshots were flaky when their light counterparts weren't. Since the dark screenshots are taken first I thought that maybe a little delay (to let the UI settle(?)) would help and it seems it does.

Reply:
Whaaaat the fuck 😆 

Mind adding an inline comment about this? 😁